frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/employee/NewApplication/index.js

```
@@ -1,153 +1,1218 @@
+        oldPropertyId: generalDetails?.oldPropertyId || prev.oldPropertyId,
+      }));
+    }
+  }, [unitDetails]);
+
+
+  useEffect(() => {
+    if (!unitDetails || unitDetails.length === 0) return;
+
+    const formattedUnits = unitDetails.map((unit) => ({
+      usageType: unit && unit.usageCategory ? unit.usageCategory : "",
+      usageFactor: unit && unit.occupancyType ? unit.occupancyType : "", // Fill if needed
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)